challenges.yaml

```diff
@@ -192,12 +192,12 @@
       Count the number of files in the current
       working directory. Print the number of
       files as a single integer.
-    example: ls | wc -l
-    example_fail: echo 2
```

Comment:
The reason I put the `example_fail` in the yaml file is so the test harness can test commands that have the same output as the correct solution, but are not correct. Needs better docs (this was a recent change).